utils/src/main/java/software/amazon/awssdk/utils/ProxySystemSetting.java

```diff
@@ -27,7 +27,13 @@ public enum ProxySystemSetting implements SystemSetting  {
     PROXY_PORT("http.proxyPort"),
     NON_PROXY_HOSTS("http.nonProxyHosts"),
     PROXY_USERNAME("http.proxyUser"),
-    PROXY_PASSWORD("http.proxyPassword")
+    PROXY_PASSWORD("http.proxyPassword"),
+
+    HTTPS_PROXY_HOST("https.proxyHost"),
+    HTTPS_PROXY_PORT("https.proxyPort"),
+    HTTPS_NON_PROXY_HOSTS("https.nonProxyHosts"),
+    HTTPS_PROXY_USERNAME("https.proxyUser"),
+    HTTPS_PROXY_PASSWORD("https.proxyPassword")
```

Comment:
I don't think we need `HTTPS_NON_PROXY_HOSTS("https.nonProxyHosts")`


https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html

>As you probably guessed these work in the exact same manner as their http counterparts, so we won't go into much detail except to mention that the default port number, this time, is 443 and that for the "non proxy hosts" list, the HTTPS protocol handler will use the same as the http handler (i.e. http.nonProxyHosts).

Reply:
Removed this enum